ext/openssl/cipher/aead/aead.c

```diff
@@ -5,16 +5,15 @@
 VALUE dOSSL;
 VALUE eCipherError;
 
-#define GetCipherInit(obj, ctx) do {                    \
-        Data_Get_Struct((obj), EVP_CIPHER_CTX, (ctx));  \
-    } while (0)
-
-#define GetCipher(obj, ctx) do {                                        \
-        GetCipherInit((obj), (ctx));                                    \
-        if (!(ctx)) {                                                   \
-            ossl_raise(rb_eRuntimeError, "Cipher not inititalized!");   \
-        }                                                               \
-    } while (0)
+#define GetCipherInit(obj, ctx) do { \
+    TypedData_Get_Struct((obj), EVP_CIPHER_CTX, RTYPEDDATA_TYPE(obj), (ctx)); \
```

Comment:
Presumably the passed `obj` is being set with `TypedData_Wrap_Struct`? Is that done by openssl?